{"name":"Lesson repository create lesson failure","fullName":"TestLessonCreateSuite/Lesson repository create lesson/TestCreate_Failure","status":"passed","statusDetails":{"message":"","trace":""},"start":1727773563261,"stop":1727773563261,"uuid":"6268fc2a-7fd4-11ef-a987-ca15b1947338","historyId":"302356b2f059cca0aea0004d53fa7880","testCaseId":"ea9457d9ae31e4848edf6aaf4acbd416","labels":[{"name":"language","value":"go1.22.5"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"h19.net58.bmstu.ru"},{"name":"thread","value":"TestLessonCreateSuite/Lesson repository create lesson/TestCreate_Failure"},{"name":"suite","value":"Lesson repository create lesson"},{"name":"package","value":"github.com/paw1a/eschool/internal/adapter/repository/postgres/test"}],"steps":[{"name":"ASSERT: Error Is","status":"passed","start":1727773563261,"stop":1727773563261,"parameters":[{"name":"Error","value":"sql: connection is already closed: persistence internal error"},{"name":"Target","value":"persistence internal error"}]}]}